### PR TITLE
fix: Join event panel opening

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/6322

https://user-images.githubusercontent.com/56407566/108744972-94482b80-7560-11eb-914e-a75d690529f2.mp4

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/controllers/public.js [hunk 1]
```diff
@@ -51,6 +51,12 @@ export default class PublicController extends Controller {
     }
   }
 
+  @action
+  async showSidePanel() {
+    await this.set('side_panel', null);
```
Comment (iamareebjamal) [inline app/controllers/public.js:56]: Why?

File: app/components/public/stream/join-video.ts [hunk 2]
```diff
@@ -32,7 +33,9 @@ export default class JoinVideo extends Component<Args> {
     this.canAccess = can_access;
   }
 
+  @action
   openPanel(): void {
+    this.args.showSidePanel();
```
Comment (iamareebjamal) [inline app/components/public/stream/join-video.ts:38]: What if showSidePanel is not passed?

What if user does not have access to the videos?

**Comment by maze-runnar** [comment]
https://user-images.githubusercontent.com/56407566/108954304-a1fdce00-7692-11eb-90d8-3ae590485d3a.mp4

**Comment by maze-runnar** [comment]
https://user-images.githubusercontent.com/56407566/108954844-5ef02a80-7693-11eb-95be-e23f892227f4.mp4

**Pull request merged**